Promote hibernate-validator to runtime scope in TCA

## Context

PR #1590 (closes #1589) promoted `hibernate-validator` from test scope to runtime in CDA and RDA by replacing the explicit test-scoped dep with `spring-boot-starter-validation`. TCA was overlooked: it already declared `spring-boot-starter-validation`, which made casual inspection look fine.

However, `trust-center-agent/pom.xml:41-45` still declares:

```xml
<dependency>
  <groupId>org.hibernate.validator</groupId>
  <artifactId>hibernate-validator</artifactId>
  <scope>test</scope>
</dependency>
```

Maven mediation: a direct declaration's scope overrides the compile-scope transitive from `spring-boot-starter-validation`. Result: **hibernate-validator is missing from the TCA runtime fat jar.**

Verified by inspecting `trust-center-agent/target/trust-center-agent.jar`:

```
BOOT-INF/lib/jakarta.validation-api-3.0.2.jar   ← only validation jar
(no hibernate-validator-*.jar)
```

## Symptom

Under `springdoc-openapi` 2.8.17 (see PR #1572), TCA's `/api/v2/openapi` hangs indefinitely. Reproduced locally on the renovate branch with curl timeout >60s. TCA log shows:

```
java.lang.NoClassDefFoundError: org/hibernate/validator/constraints/Range
  at org.springdoc.core.utils.SchemaUtils.lambda$applyValidationsToSchema$3(SchemaUtils.java:294)
  at org.springdoc.core.utils.SchemaUtils.applyValidationsToSchema(SchemaUtils.java:228)
  ...
Caused by: java.lang.ClassNotFoundException: org.hibernate.validator.constraints.Range
```

`SchemaUtils.applyValidationsToSchema` unconditionally references `org.hibernate.validator.constraints.Range`. The `NoClassDefFoundError` is an `Error` subclass, which Reactor's `throwIfFatal` logs and then drops the signal chain — the HTTP response never completes, so the client hangs. This is what caused the Pages CI job to run 3h+ until cancelled on PR #1572.

## Verification

1. Checkout `renovate/org.springdoc-springdoc-openapi-starter-webflux-ui-2.x`.
2. Build TCA, start with keystore (valkey) running: `/api/v2/openapi` → hang + `NoClassDefFoundError` in log.
3. Remove `<scope>test</scope>` from the `hibernate-validator` dep, rebuild.
4. `curl http://localhost:8080/api/v2/openapi` → HTTP 200 in <1s, 4510 bytes.

## Fix

In `trust-center-agent/pom.xml`, drop `<scope>test</scope>` from the `hibernate-validator` declaration (or remove the declaration entirely — `spring-boot-starter-validation` pulls it transitively at compile scope). Same fix pattern as #1590.

## Related

- #1589 / #1590 — CDA + RDA equivalent
- #1572 — springdoc 2.8.17 upgrade, currently blocked by this bug

--- a/trust-center-agent/pom.xml
+++ b/trust-center-agent/pom.xml
@@ -34,17 +34,6 @@
     </dependency>
 
     <dependency>
-      <groupId>jakarta.validation</groupId>
-      <artifactId>jakarta.validation-api</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>org.hibernate.validator</groupId>
-      <artifactId>hibernate-validator</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-validation</artifactId>
     </dependency>

